corehq/apps/userreports/reports/filters.py

```diff
@@ -39,6 +39,26 @@ def to_sql_values(self):
         }
 
 
+class NumericFilterValue(FilterValue):
+
+    def __init__(self, filter, value):
+        assert filter.type == "numeric"
+        assert (isinstance(value, dict) and "operator" in value and "operand" in value) or value is None
+        super(NumericFilterValue, self).__init__(filter, value)
+
+    def to_sql_filter(self):
+        if self.value is None:
+            return ""
+        return "{0} {1} :operand".format(self.filter.field, self.value['operator'])
```

Comment:
yeah i saw that, but i think if someone else uses this class it'd be good for the assert to be here